src/zenml/log_stores/otel/otel_flavor.py

```diff
@@ -0,0 +1,125 @@
+#  Copyright (c) ZenML GmbH 2025. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""OpenTelemetry log store flavor."""
+
+from typing import Type
+
+from pydantic import Field
+
+from zenml import __version__
+from zenml.enums import StackComponentType
+from zenml.log_stores import BaseLogStore, BaseLogStoreConfig
+from zenml.stack.flavor import Flavor
+
+
+class OtelLogStoreConfig(BaseLogStoreConfig):
+    """Configuration for OpenTelemetry log store.
+
+    Attributes:
+        service_name: Name of the service (defaults to "zenml").
+        max_queue_size: Maximum queue size for batch processor.
+        schedule_delay_millis: Delay between batch exports in milliseconds.
+        max_export_batch_size: Maximum batch size for exports.
+    """
+
+    service_name: str = Field(
+        default="zenml",
+        description="Name of the service for telemetry",
+    )
+    service_version: str = Field(
+        default=__version__,
+        description="Version of the service for telemetry",
+    )
+    max_queue_size: int = Field(
+        default=8096,
+        description="Maximum queue size for batch log processor",
+    )
+    schedule_delay_millis: int = Field(
+        default=15000,
+        description="Export interval in milliseconds",
+    )
+    max_export_batch_size: int = Field(
+        default=512,
+        description="Maximum batch size for exports",
+    )
+
+
+class OtelLogStoreFlavor(Flavor):
+    """OpenTelemetry log store flavor."""
+
+    @property
+    def name(self) -> str:
+        """Name of the flavor.
+
+        Returns:
+            The name of the flavor.
+        """
+        return "otel"
+
+    @property
+    def docs_url(self) -> str:
+        """URL to the flavor documentation.
+
+        Returns:
+            The URL to the flavor documentation.
+        """
+        return "https://docs.zenml.io/stack-components/log-stores/otel"
+
+    @property
+    def sdk_docs_url(self) -> str:
+        """URL to the SDK docs for this flavor.
+
+        Returns:
+            The URL to the SDK docs for this flavor.
+        """
+        return self.docs_url
+
+    @property
+    def logo_url(self) -> str:
+        """URL to the flavor logo.
+
+        Returns:
+            The URL to the flavor logo.
+        """
+        # TODO: Add a logo for the OpenTelemetry log store
```

Comment:
we would never really need this logo, because this flavor is never actually directly used.